exporter/elasticsearchexporter/README.md

```diff
@@ -82,14 +82,21 @@ As a consequence of supporting [confighttp], the Elasticsearch exporter also sup
 The Elasticsearch exporter sets `timeout` (HTTP request timeout) to 90s by default.
 All other defaults are as defined by [confighttp].
 
-### Queuing
+### Queuing and batching
 
-The Elasticsearch exporter supports the common [`sending_queue` settings][exporterhelper]. However, the sending queue is currently disabled by default.
+The Elasticsearch exporter supports the common [`sending_queue` settings][exporterhelper] which supports both queueing and batching. However, the sending queue is currently disabled by default. Sending queue can be enabled by setting `sending_queue::enabled` to `true`. The batching support in sending queue is also disabled by default. Batching can be enabled by defining `sending_queue::batch`.
+
+The exporter will perform its own buffering and batching and will issue async requests to Elasticsearch in all cases other than if any of the following conditions are met:
+
+- `sending_queue::batch` is defined (irrespective of `sending_queue` being enabled or not)
+- `batcher::enabled` is defined (set to `true` or `false`)
+
+NOTE: Exporter's inbuilt handling of batching is a legacy feature and is scheduled to be deprecated when `sending_queue` API is out of the experimental stage. The best way to enable this mode is to remove all `batcher` and `sending_queue::batch` configurations.
 
 ### Batching
 
 > [!WARNING]
-> The `batcher` config is experimental and may change without notice.
+> The `batcher` config is now deprecated and will be removed in an upcoming version. Check the [queueing and batching](#queueing-and-batching) section for using the `sending_queue` setting that supersedes `batcher`. In the interim, `batcher` configurations are still valid, however, they will be ignored if `sending_queue::batch` is defined even if `sending_queue` is not enabled.
```

Comment:
What happens if `sending_queue::batch` is set but `sending_queue::enabled` is set to `false`, will it also be ignored?